oeps/oep-2.rst

```diff
@@ -0,0 +1,100 @@
+==========================
+OEP-2: Repository Metadata
+==========================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`oep-2`                              |
++---------------+-------------------------------------------+
+| Title         | Repository Metadata                       |
++---------------+-------------------------------------------+
+| Last-Modified | 2016-04-21                                |
++---------------+-------------------------------------------+
+| Author        | Calen Pennington <cale@edx.org>           |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+| Created       | 2016-04-21                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+Proposes a standard format for repository metadata for Open edX projects.
+
+Specification
+=============
+
+Each repo in the Open edX system will include a file ``openedx.yml``, with the
```

Comment:
Fundamentally, I think this file is Open edX specific, and thus should have that in the name. Once it's there, I'm not sure there's much to be gained from making the name longer.